src/Symfony/Component/Messenger/Exception/TransportException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Exception;
+
+/**
+ * @author Eric Masoero <em@studeal.fr>
+ *
+ * @experimental in 4.2
```

Comment:
Is this message correct? This is going to be merged in `master`, which is `4.3`. Thanks.

Reply:
It still says 4.2 in other files

Reply:
Ahah true x)
I bump it when I'll be sure what to do with the exception message :)

Reply:
bumping will be done in another PR before releasing 4.3